### DISCUSSION TITLE
Is a DAO obligated in some way to execute the result of a vote?

### DISCUSSION THREAD

**Discussion by jonathan-melnik** [snapshot-labs/snapshot#1372] [Q&A] [answered]
I'm missing a part of the puzzle with DAOs.

After a vote is casted, is there some system in place that performs the action automatically?

**Comment by metrox-eth** [comment, marked as answer]
Hi yes, you can use SafeSnap for that, have a look here: https://blog.gnosis.pm/introducing-safesnap-the-first-in-a-decentralized-governance-tool-suite-for-the-gnosis-safe-ea67eb95c34f & https://gnosis.github.io/zodiac/docs/tutorial-module-reality/get-started/

We're also really excited because very soon we'll allow spaces to send their votes to starknet, a L2, and then that'll be able to directly execute on chain. Coming around January 2022!
